ansible/roles/esdc-mon/templates/es-post-deploy.sh.j2

```diff
@@ -25,20 +25,25 @@ zbx() {
 	return "${e}"
 }
 
+log "Populating authorized_keys"
```

Comment:
Please add a comment with a simple explanation why we are doing this.

(A link to `https://github.com/erigones/esdc-factory/issues/43` should be enough)